packages/tooltip/src/LionTooltip.js

```diff
@@ -66,6 +66,15 @@ export class LionTooltip extends ArrowMixin(OverlayMixin(LitElement)) {
     });
   }
 
+  _hasDisabledInvoker() {
+    return (
+      this._overlayCtrl &&
+      this._overlayCtrl.invoker &&
+      /** @type {HTMLElement & { disabled: boolean }} */ ((this._overlayCtrl.invoker).disabled ||
+        this._overlayCtrl.invoker.getAttribute('aria-disabled') === 'true')
+    );
+  }
```

Comment:
```suggestion
  _hasDisabledInvoker() {
    return (
      this._overlayCtrl &&
      this._overlayCtrl.invoker &&
      (/** @type {HTMLElement & { disabled: boolean }} */ (this._overlayCtrl.invoker).disabled ||
        this._overlayCtrl.invoker.getAttribute('aria-disabled') === 'true')
    );
  }
```

should fix the typecast

Reply:
The linter says otherwise..